internal/provider/provider.go

```diff
@@ -90,16 +95,21 @@ func (p *ReplicatedProvider) Configure(ctx context.Context, req provider.Configu
 
 	httpClient := platformclient.NewHTTPClient(apiOrigin, apiToken)
 	kotsAPI := &kotsclient.VendorV3Client{HTTPClient: *httpClient}
+	customersAPIClient := NewCustomersAPIClient(apiToken)
+
+	clients := ReplicatedProviderClients{
+		kotsVendorV3Client: *kotsAPI,
+		customersAPIClient: *customersAPIClient,
+	}
 
-	// VendorApi client configuration.
-	client := kotsAPI
-	resp.DataSourceData = client
-	resp.ResourceData = client
+	resp.DataSourceData = &clients
+	resp.ResourceData = &clients
```

Comment:
The kotsAPI from [replicated cli](https://github.com/replicatedhq/replicated) also has the capabilities to manage clusters

Reply:
Yes I know but it's missing some things that made it impossible to use it for the provider. For instance, we miss a simple get method for a customer returning all available properties in the API. We already checked with your team the possibility of using the CLI and it didn't fit the needs of the provider 😓